sg_AbstractSpatAlgorithm.cpp

```diff
@@ -142,6 +142,9 @@ std::unique_ptr<AbstractSpatAlgorithm> AbstractSpatAlgorithm::make(SpeakerSetup
             return MbapSpatAlgorithm::make(speakerSetup, sources.getKeys());
         }
     case SpatMode::hybrid:
+        // Parallel Hybrid mode is much worse than regular hybrid mode.
```

Comment:
interesting, do we know why ? and is it like, a constant performance issue or only with small buffer sizes?